AgentDiscoveries-Backend/src/main/resources/db/migration/V7_add_location_report_title.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE location_reports
+ADD title mediumtext;
```

Comment:
Did you choose mediumtext specifically? What are the advantages of using mediumtext over other types, like varchar(255)?

Reply:
No, I changed the type to Varchar(255)